.teamcity/_Self/buildTypes/PublishPackageToAzure.kt

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+package _self.buildTypes
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.swabra
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.finishBuildTrigger
+
+object PublishPackageToAzure : BuildType ({
+    name = "Publish Package to Azure"
+    description = "Publishes NuGet package to the Azure feed"
+
+    publishArtifacts = PublishMode.SUCCESSFUL
+
+    option("shouldFailBuildOnAnyErrorMessage", "true")
+
+    params {
+        param("nuGet.packageFile", "placeholder")
+        param("nuGet.packageVersion", "placeholder")
```

Comment:
Please add this parameter:
```suggestion
        param("nuGet.packageVersion", "placeholder")
        param("env.VSS_NUGET_EXTERNAL_FEED_ENDPOINTS",'{"endpointCredentials": [{"endpoint": "%azureArtifacts.feed.nuget%","username": "%azureArtifacts.edFiBuildAgent.userName%","password": "%azureArtifacts.edFiBuildAgent.accessToken%"}]}')
```

Reply:
@stephenfuqua 
Updated PublishToAzure kt file with env.VSS_NUGET_EXTERNAL_FEED_ENDPOINTS param. Please help review and merge the changes.